ansible/roles/software/datagov-deploy-fgdc2iso/defaults/main.yml

```diff
@@ -0,0 +1,3 @@
+---
+fdgc2iso_tls_certificate_filepath: /etc/ssl/certs/datagov_host.crt
+fgdc2iso_tls_key_filepath: /etc/ssl/private/datagov_host.key
```

Comment:
Since these don't really make sense as defaults and we're not actually managing these files, I would just make them required variables for the role, e.g. assert they are defined within the tasks at runtime and not include them in defaults.

Reply:
ok, but this means we need to repetitively define the same variable for all three environments?

Reply:
Or you could define them once as a [playbook group variable](https://github.com/GSA/datagov-deploy/blob/develop/ansible/group_vars/all/vars.yml).

Reply:
Oh yeah. I thought there got to be a better way than declaring for all envs.